<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>464</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="1" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Icona</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="3">
      <widget class="QComboBox" name="applicationTypeEntry">
       <item>
        <property name="text">
         <string>Applicazione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Link</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Directory</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="17" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Commento</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="commandEntry"/>
     </item>
     <item row="17" column="3">
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Apri</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Versione</string>
       </property>
      </widget>
     </item>
     <item row="17" column="4">
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Salva</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="nameEntry"/>
     </item>
     <item row="17" column="5">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="3">
      <widget class="QLineEdit" name="iconEntry"/>
     </item>
     <item row="14" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="1" colspan="3">
      <widget class="QCheckBox" name="terminalCheckBox">
       <property name="text">
        <string>Apri nel terminale</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLabel" name="Comando">
       <property name="text">
        <string>Comando</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QLineEdit" name="commentEntry"/>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="versionEntry">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1" colspan="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="1" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Tipo</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Esporta</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="3">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
